core/embed/rust/src/ui/layout/obj.rs

```diff
@@ -376,12 +366,12 @@ extern "C" fn ui_layout_touch_event(n_args: usize, args: *const Obj) -> Obj {
     unsafe { util::try_with_args_and_kwargs(n_args, args, &Map::EMPTY, block) }
 }
 
-#[cfg(not(feature = "model_tt"))]
+#[cfg(all(feature = "buttons", not(feature = "touch")))]
```

Comment:
why is `buttons` here?

Reply:
Right, just simple `#[cfg(feature = "touch")]` and `#[cfg(not(feature = "touch"))]` works - the same below with "buttons"

Done in [d877e72](https://github.com/trezor/trezor-firmware/pull/2262/commits/d877e725bcb5b071562e0cfd469091a49affe7b0)